Apple-TV/VLCAboutViewController.m

```diff
@@ -65,9 +65,8 @@ - (void) viewDidDisappear:(BOOL)animated
     [self stopAnimation];
 }
 
-- (UIView *)preferredFocusedView
-{
-    return self.blablaTextView;
+- (NSArray<id<UIFocusEnvironment>> *)preferredFocusEnvironments {
+    return @[self.blablaTextView];
```

Comment:
wow blablaTextView ??

Reply:
Good point, this might need indeed a renaming.